tests/foreman/api/test_errata.py

```diff
@@ -55,6 +54,8 @@
 
 CUSTOM_REPO_URL = FAKE_6_YUM_REPO
 CUSTOM_REPO_ERRATA_ID = FAKE_2_ERRATA_ID
+DISTRO_RHEL6 = settings.distro.image_el6
```

Comment:
yeah, sure will provide test results, I already did test for feasibility sake and it worked. But since then did many changes to the code, so feel, I should test it once again.